src/en/charms-offline-config.md

```diff
@@ -50,7 +49,10 @@ and the appropriate Juju configuration changes (covered
    The **client** requires access to the backing cloud in order to create a
    controller. Most public clouds have a RESTful API that operates over TCP
    port 443. A special case is the localhost cloud, in which the client talks
-   to the local LXD daemon.
+   to the local LXD daemon.  
+
+    Depending on the provider chosen and the network design the **controller**
```

Comment:
there may be a wayward space before *Depending*...

Reply:
It's intended. It's how to get that paragraph indented with the upper paragraph.